.github/workflows/update-snyk.yaml

```diff
@@ -0,0 +1,26 @@
+name: Snyk report update
+on:
+  schedule:
+    - cron: '0 0 * * 0' # midnight every Sunday
+jobs:
+  snyk-report:
+    name: Update Snyk report in the docs directory
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build reports
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        run: |
+          stable_release="release-$(sed -E 's/\.[0-9]+$//g' VERSION)"
+          git checkout "$stable_release"
+          make snyk-report
```

Comment:
Shall the cronjob run off the stable release branch or off `master`?

I think it should _run_ off the `master` branch (so we don't have to cherry-pick changes to the script), and store the results in the stable branch. :thinking:

Reply:
Ah wow, good point. Will fix.